tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -253,6 +253,24 @@ def test_create_from_snapshot_with_env(self):
     self.assertEqual(req.env, ('VAR', 'VAL'))
 
 
+class TestInputFileCreation(TestBase, unittest.TestCase):
+  def test_input_file_creation(self):
+    file_name = 'some.filename'
+    file_contents = b'some file contents'
+
+    input_file = InputFileContent(path=file_name, content=file_contents)
+    digest = self.scheduler.product_request(Digest, [input_file])[0]
```

Comment:
```suggestion
    digest, = self.scheduler.product_request(Digest, [input_file])
```
I try to do this because it will also check if there happens to be any more than just one value, which may happen e.g. if the method's API changes for some reason.

Reply:
changed